index.html

```diff
@@ -12,21 +12,52 @@
     <div id="signin_background">
         <h1>Sign in</h1>
 
-        <form class="info-user">
-            <label for="username">Username:</label>
+    <div id="signin-form">
+        <h1>Connection</h1>
+        <form class="InfoUser">
+            <label for="username">Nom d'utilisateur:</label>
             <br>
-            <input type="text" id="username" name="username" placeholder="Enter Username:"><br><br>
+            <input type="text" id="signin-username" name="signin-username"
+                placeholder="Entrez nom d'utilisateur:"><br><br>
```

Comment:
Ne pas utiliser de \<br>!
```suggestion
                placeholder="Entrez nom d'utilisateur:">
```